### ISSUE TITLE
 Chat - Download confirmation window still dark in light theme

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#33120]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.13-0
**Reproducible in staging?:** Y
**Reproducible in production?:** N 
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Open the app
2. Login with any account
3. Change the theme for light (Settings -> Preferences -> Theme -> Light)
4. Go to any chat with attachment -> download it

## Expected Result:
The confirmation window is light in light mode

## Actual Result:
The confirmation window is dark in light mode
The same with the downloaded share code

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

![Bug6313481_1702579787104!Screenshot_2023-12-14-19-16-13-69_4f9154176b47c00da84e32064abf1c48](https://github.com/Expensify/App/assets/78819774/4d50abcc-31e9-468c-9269-c84908c563bd)

![Bug6313481_1702579787120!Screenshot_2023-12-14-19-10-37-14_4f9154176b47c00da84e32064abf1c48](https://github.com/Expensify/App/assets/78819774/67cff726-3dfd-441b-b2e5-7a0b7dfc3d68)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by grgia** [comment]
@dubielzyk-expensify is this a bug?

**Comment by grgia** [comment]
@joelbettner I'll take this one

**Comment by dubielzyk-expensify** [comment]
The download confirmation uses a system modal/dialog with a system scrim/overlay, so no. On iOS it also uses system dialog:
![CB2D8864-DEDA-4DA4-BCD4-DE157818C2C3](https://github.com/Expensify/App/assets/145416827/bcc8788f-07b8-4c6b-9fa2-9813464c273a)

So unless we wanna replace this with our own dialog, then this isn't a bug.

**Comment by grgia** [comment]
@situchan good call. But this screenshot in the description isn't a native modal, it's using our app colors

**Comment by grgia** [comment]
https://github.com/Expensify/App/blob/1b58d05718ba57d20696f5d1f9fb404caeecf15d/src/libs/fileDownload/FileUtils.ts#L8-L23

**Comment by grgia** [comment]
https://github.com/Expensify/App/blob/914fe828aaf772739f02c8a8eff9f12428027b66/android/app/src/main/res/values/styles.xml#L30-L37

**Comment by grgia** [comment]
Either way, I don't really mind it being dark here

**Comment by dubielzyk-expensify** [comment]
Ahh, I see now. This is only a Android bug then cause we don't customize the iOS dialog with our colors. 

For Android, let's fix it to respect the theme colors if that's doable

**Comment by jasperhuangg** [comment]
This seems like a dupe (not the same flow but definitely uses the same components): https://github.com/Expensify/App/issues/33138
